scala-thoughts.md

```diff
@@ -0,0 +1,19 @@
+I was initially pretty unimpressed with Scala. Sure, string interpolation is pretty cool,
+and I can (now, at least) appreciate the "immutable/functional first" philosophy; but syntax-wise, I initially treated Scala much like Java
+(because Scala let's you get away with it--more on this later), and so I was very much displeased by what I perceived to be a design decision
+to maintain arguably overbearing syntax--particularly when compared to other languages to which I'm partial such as Haskell and Python.
+This concept bugged me so much that I looked up the style guide and found something to my liking:
+you don't, in fact, need to use a dot operator in many cases, and it is even frowned upon in some cases.
+Not only did this make me appreciate Scala a little more, but I think it was a good design decision given its name derives from "scalable language."
+A scalable language should be flexible in its syntax, and Scala's syntax is flexible on so many levels:
+Scala supports unicode, operator overloading, and various ways of calling built-in (and presumably other) functions
+via the ommission of periods and parentheses.
+
```

Comment:
I notice that you omitted the parenteses and periods more than the other solution I looked at (and my own); it's really interesting to see how different programming styles Scala supports.